<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatSealModuleWidget</class>
 <widget class="QWidget" name="HeatSealModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1720</width>
    <height>998</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="moduleWidget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>热封调试</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLiquidDetect">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>托盘推出</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnModuleReset">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>模块复位</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>213</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>托盘运动</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;托盘推出位置：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>213</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Y轴复位</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>目标温度：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Z轴下压位置：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLiquidDetect_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>托盘推入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>274</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Z轴运动</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>274</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Z轴复位</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;mm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>280</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;mm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>℃</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>70</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开始热封</string>
      </property>
     </widget>
     <widget class="SwitchButton" name="swControlDuct" native="true">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>490</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLiquidDetect_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>获取参数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLiquidDetect_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>保存参数</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;温度补偿IN：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>400</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;温度补偿OUT：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>400</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="editTargetTemp">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>490</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始加热</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLiquidDetect_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>480</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>读取当前温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>490</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;当前温度OUT：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>490</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>575</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>未检测</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLiquidDetect_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>570</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>在位检测</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header location="global">hcscitoolkits/switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
